cpp/src/neighbors/faiss_select/DistanceUtils.h

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I'm a little surprised that this didn't cause an error from our copyright-verify script https://github.com/rapidsai/cuvs/blob/43ed8ccd9bf2a7f6297d396db36f1a0795a66ba7/.pre-commit-config.yaml#L102-L112 (iirc, I had to manually exclude these files in RAFT with the previous copyright checker). @KyleFromNVIDIA is this what you'd expect to happen here?